src/comments.js

```diff
@@ -933,6 +938,38 @@ function printComment(commentPath, options) {
   }
 }
 
+function isJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+  return (
+    lines.length > 1 &&
+    lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*")
+  );
+}
+
+function printJsDocComment(comment, options) {
+  const lines = comment.value.split("\n");
+  const tabWidth = options.tabWidth;
+  const indentSize = util.getIndentSize(comment.value, tabWidth);
+
+  const parts = [];
+
+  lines.forEach((line, index) => {
+    parts.push(
+      docBuilders.align(
+        indentSize,
+        // We only want to start with a leading space if we are not on
+        // the first line of a JSDoc comment.
+        (index > 0 ? " " : "") + line.trimLeft()
+      )
+    );
+    if (index < lines.length - 1) {
+      parts.push(docBuilders.hardline);
+    }
```

Comment:
It would be nicer to `const parts = lines.map(......); return `join(hardline, parts);`